<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DomainLearningWindow</class>
 <widget class="QMainWindow" name="DomainLearningWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DomainLearningWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Scenes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="scenes_list"/>
       </item>
       <item>
        <widget class="QCheckBox" name="track_objects_checkbox">
         <property name="text">
          <string>Track Objects</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="calibrate_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Calibrate Projector</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="init_scene_size_btn">
         <property name="text">
          <string>Init Scene Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="project_name_le">
         <property name="text">
          <string>[project_name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_proj_btn">
         <property name="text">
          <string>Load...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>90</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="instructions_label">
         <property name="text">
          <string>Instructions ... / Status ... </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate_Project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="actionSave_Project"/>
   </widget>
   <widget class="QMenu" name="menuEvents_and_Actions">
    <property name="title">
     <string>Events and Actions</string>
    </property>
    <addaction name="actionEvents"/>
    <addaction name="actionActions"/>
    <addaction name="actionRules"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEvents_and_Actions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_Project">
   <property name="text">
    <string>Create Project ...</string>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project ...</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project ...</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="text">
    <string>Events ...</string>
   </property>
  </action>
  <action name="actionActions">
   <property name="text">
    <string>Actions ...</string>
   </property>
  </action>
  <action name="actionRules">
   <property name="text">
    <string>Rules ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
